### ISSUE TITLE
imatlab_export_fig('print-svg')

### ISSUE THREAD

**Issue by GorgiAstro** [imatlab/imatlab#50]
Hi,
I really love the capabilities of imatlab, thanks a lot.
When exporting the notebook to LaTeX, I would like to have the inline figures saved as svg. Is it possible? Does it only involve adding a case to imatlab_export_fig.m with `print(children(i), exported{i}, '-dsvg', '-r0')` ?
Thank you,
Best regards
Yzokras

**Comment by anntzer** [comment]
Pushed support for SVG export.
The notebook seems not to like it when trying to export to tex though ("nbconvert failed: Object of type bytes is not JSON serializable"); but I feel like the issue is on nbconvert's side?

**Comment by GorgiAstro** [comment]
Thanks a lot. For me, it first threw the error "nbconvert failed: Inkscape svg to pdf conversion failed".
But after installing Inkscape, it works, the SVG figure gets exported to pdf. I run Ubuntu 18.10.

By the way, I looked at how to make Matlab print directly a PDF, do you think it's doable?
- Matlab's `print` function can do it
- Can Jupyter display it?
- In the method `_export_figures`, how to format the PDF data?

**Comment by anntzer** [comment]
I don't know if Jupyter supports it.  Why don't you try/look it up yourself and open a PR accordingly :-)?
*If* Jupyter supports it (unclear...), it's likely just a matter of calling `_send_display_data` with the right mimetype.

**Comment by GorgiAstro** [comment]
You're right, I'll look it up myself. I already cloned the repo, I just wanted to know whether you had an obvious solution before I spend too long on this :)
